src/App.js

```diff
@@ -1,19 +1,30 @@
-import React from 'react'
+import React, { useState } from 'react'
 import './App.css';
 import Header from './Components/Header'
 import PosterGrid from './Components/PosterGrid'
 import SelectedMovie from './Components/SelectedMovie'
-import MovieData from './MockData/MovieData'
+// import MovieData from './MockData/MovieData'
+import Spinner from './Components/Spinner'
 
 class App extends React.Component {
   constructor() {
     super()
     this.state = {
-      movies: MovieData,
+      movies: null,
```

Comment:
Assigning 'null' to 'movies' & 'singleMovie might create some bugs in your code. The best practice is always to assign the data type that you're expecting to receive from the API. Is 'movie' an object, array, number, or string?